atomist/get-started.md

```diff
@@ -0,0 +1,381 @@
+     step 2.
+
+8. Click **Save Configuration**. Atomist will test the connection. You'll see
+   some green check marks when a connection has been established.
+
+   ![GAR configuration successful](./images/gar/config_success.png){:
+   width="700px"}
+
+   Next, we will create a new PubSub subscription on the `gcr` topic in Google
+   Artifact Registry. This subscription will notify Atomist when new images are
+   pushed to the registry.
+
+9. Copy the URL in the **GCR Events Webhook** field to your clipboard. This will
+   be the `PUSH_ENDPOINT_URI` for the PubSub subscription.
+
+10. Define the following three variable values, in addition to the `PROJECT_ID`
+    and `SERVICE_ACCOUNT_ID` from earlier:
+
+    - `PUSH_ENDPOINT_URL`: the webhook URL copied from the Atomist workspace.
+    - `SERVICE_ACCOUNT_EMAIL`: the service account address; a combination of the
+      service account ID and project ID.
+    - `SUBSCRIPTION`: the name of the PubSub to be created (can be anything).
+
+    ```bash
+    PUSH_ENDPOINT_URI={COPY_THIS_FROM_ATOMIST}
+    SERVICE_ACCOUNT_EMAIL="${SERVICE_ACCOUNT_ID}@${PROJECT_ID}.iam.gserviceaccount.com"
+    SUBSCRIPTION="atomist-gar-integration-subscription"
+    ```
+
+11. Create the PubSub for the `gcr` topic.
+
+    ```bash
+    gcloud pubsub subscriptions create ${SUBSCRIPTION} \
+      --topic='gcr' \
+      --push-auth-token-audience='atomist' \
+      --push-auth-service-account="${SERVICE_ACCOUNT_EMAIL}" \
+      --push-endpoint="${PUSH_ENDPOINT_URI}"
+    ```
+
+When the first image push is successfully detected, a green check mark on the
+integration page will indicate that the webhook event was received and that the
+integration works.
+
+</div>
+<div id="tab-ghcr" class="tab-pane fade" markdown="1">
+<!-- GitHub Container Registry -->
+
+To integrate Atomist with GitHub Container Registry, connect your GitHub
+account, and enter a personal access token for Atomist to use when pulling
+container images.
+
+1. Go to <https://dso.docker.com> and sign in using your Docker ID credentials.
+2. Connect your GitHub account as instructed in the
+   [GitHub app page](./integrate/github.md#connect-to-github). Install the app
+   into the GitHub account that contains your GitHub Container Registry.
+3. Open the [**Integrations**](https://dso.docker.com/r/auth/integrations){:
+   target="blank" rel="noopener" class=""} tab, and click the **Configure** link
+   next to the **GitHub Container Registry** in the list of integrations.
+4. Fill out the fields and click **Save Configuration**.
+
+   The **Personal access token** is required for scanning images in private
+   repositories. The token must have the
+   [`read:packages` scope](https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages).
+
+   You can leave the **Personal access token** field blank if you only want to
+   scan images in public repositories.
+
+</div>
+<hr>
+</div>
+
+## Link images to Git repository
```

Comment:
We should probably document the opt-in in buildx here as well. See https://github.com/docker/buildx/issues/1290